dist/index.js

```diff
@@ -140,13 +147,27 @@ function getInput(name, pattern, defaultValue) {
  * @returns {Promise<string>} The directory SAM CLI is installed in.
  */
 // TODO: Support more platforms
-// TODO: Support caching
 async function installUsingNativeInstaller(version) {
   if (os.platform() !== "linux" || os.arch() !== "x64") {
     core.setFailed("Only Linux x86-64 is supported with use-installer: true");
     return "";
   }
 
+  // Must be full semantic version; downloads version directly from GitHub
+  if (version && !isSemver(version)) {
```

Comment:
what if `1.2.*` is used

Reply:
Not supported.

Reply:
can we resolve the version and locate the latest one before downloading?

Reply:
Not sure what you mean, but checking latest and cache is in the TODOs.

Reply:
when `1.2.*` is specified, we first list all the versions and locate the latest version that satisfy `1.2.*`. Then check if that version is cached.

Reply:
I imagine most customers want to use `1.*`

Reply:
(can be in a follow up)

Reply:
Yeah, not sure how important that is. The whole `*` thing is an implementation detail anyway and has never been advertised, it's just been "SAM CLI version".